tests/ui/postgres/deprecated_rename.stderr

```diff
@@ -0,0 +1,13 @@
+error: trybuild test needs to fail for stderr checking
+  --> $DIR/deprecated_rename.rs:10:5
+   |
+10 |     compile_error!("trybuild test needs to fail for stderr checking");
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+warning: use of deprecated function `sqlx::_rename`: sqlx(rename) is now called sqlx(type_name)
```

Comment:
Eh, this isn't _great_. In our experience, people get confused easily if the error messages aren't direct enough. I would make sure to be clear we're talking about `#[sqlx(rename = "...")]` and `#[sqlx(type_name = "...")]` if we can, otherwise it'd just be better to emit an error.

It's nice that it's pointing at the correct span at least.